microsoft/PowerToys
# [build] stop building PowerRenameUWPUI

## Summary of the Pull Request

**What is this about:**
The PowerRenameUWPUI project was needed when PowerToys supported the MSIX installer.

**What is include in the PR:** 
Remove the project from the build dependencies.

**How does someone test / validate:** 
Build the solution and verify that PowerRenameUWPUI project doesn't add any file to `x64/[Debug/Release]/`
One project shows as skipped in the build output:
```
========== Build: 82 succeeded, 0 failed, 0 up-to-date, 1 skipped ==========
```

## Quality Checklist


## Contributor License Agreement (CLA)
A CLA must be signed. If not, go over [here](https://cla.opensource.microsoft.com/microsoft/PowerToys) and sign the CLA.
